test/testdrive/kafka-exactly-once-sink.td

```diff
@@ -300,6 +316,12 @@ For FORMAT JSON, you need to manually specify an Avro consistency topic using 'C
 $ kafka-verify format=json sink=materialize.public.json_reuse_topic sort-messages=true key=false
 {"before": null, "after": {"row": {"column1": 1, "b": 2, "column3": 3}}, "transaction": {"id": "0"}}
 
+> CREATE SINK json_reuse_topic_2 FROM rt_binding_consistency_test_source
```

Comment:
Inexplicably github won't let me comment on an arbitrary line in the file, but: the test above this one (json_reuse_topic) could use a biiiiig warning label/comment about how it's actually creating a nonced topic since it's testing the non-parenthesized format. I would also rename the sink name to something like `json_reuse_topic_no_parens` to make it very clear

Reply:
I renamed and put in a little note!